snapcraft/_store.py

```diff
@@ -272,12 +274,93 @@ def download(snap_name, channel, download_path, arch):
     try:
         with _requires_login():
             store.download(snap_name, channel, download_path, arch)
-    except storeapi.errors.SnapNotFoundError:
-        raise RuntimeError(
-            'Snap {name} for {arch} cannot be found'
-            ' in the {channel} channel'.format(name=snap_name, arch=arch,
-                                               channel=channel))
```

Comment:
this code is gone because SnapNotFoundError will be raised to top-level exception handler with equivalent info, right ?